src/main.py

```diff
@@ -161,7 +161,7 @@ async def stop(interaction: Interaction) -> None:
         await interaction.response.send_message("Nothing is playing.", ephemeral=True)
```

Comment:
There's a couple missing periods still at https://github.com/anoadragon453/busty/blob/8ca62a58a34faa6ec996e6e6515b525f92631bf4/src/main.py#L196

https://github.com/anoadragon453/busty/blob/8ca62a58a34faa6ec996e6e6515b525f92631bf4/src/main.py#L214

(and you may want to remove the colons as well as you did below.)